apps/routes/lib/route.ex

```diff
@@ -29,7 +30,8 @@ defmodule Routes.Route do
           direction_destinations: %{0 => String.t(), 1 => String.t()} | :unknown,
           description: gtfs_route_desc,
           fare_class: gtfs_fare_class,
-          custom_route?: boolean
+          custom_route?: boolean,
+          line: []
```

Comment:
FYI you end up adding the item to this property (as opposed to a list as suggested by `line: []`), so the resulting struct ends up being like:

```
%Routes.Route{
  color: "DA291C",
  custom_route?: false,
  description: :rapid_transit,
  direction_destinations: %{
    0 => "Ashmont/Braintree", 
    1 => "Alewife"
  },
  direction_names: %{
    0 => "Southbound",
    1 => "Northbound"
  },
  fare_class: :rapid_transit_fare,
  id: "Red",
  line: %JsonApi.Item{
    attributes: nil,
    id: "line-Red",
    relationships: nil,
    type: "line"
  },
  long_name: "Red Line",
  name: "Red Line",
  sort_order: 10010,
  type: 1
}
```

Reply:
**suggestion:** change to `line_id: String.t()` and assign it the line's ID instead of including the entire `%JsonApi.Item{}`.

Reply:
that'll also fix a backend error I'm noticing because the application doesn't know how to encode the `%JsonApi.Item{}` struct to return it in a JSON format to the frontend. better to extract the information we need from that struct into a more concise format.

<img width="338" alt="image" src="https://github.com/mbta/dotcom/assets/2136286/8374f6a1-c7f7-498f-b0de-957f5cebf8aa">